static/tabs/instructions.html

```diff
@@ -0,0 +1,39 @@
+<div class="tab-pane fade" id="instructions" role="tabpanel" aria-labelledby="instructions-tab">
+    <div class="container mb-4">
+        <div class="row g-3">
+            <div class="col-lg-6">
+                <h2>Connecting PrusaSlicer/SuperSlicer</h2>
```

Comment:
Semantically this should be an h1 as mentioned in other pr.

Reply:
Fixed :white_check_mark: